Update Open WebUI Open WebUI LXC to v0.5.20 causes script to Error out

### ✅ Have you read and understood the above guidelines?

Yes

### 📜 What is the name of the script you are using?

Open WebUI Open WebUI LXC

### 📂 What was the exact command used to execute the script?

update

### 📝 Provide a clear and concise description of the issue.

I run the update command and it fails out with this error:
```
⠼ Updating Open WebUI (Patience)error: Your local changes to the following files would be overwritten by merge:
        backend/open_webui/static/site.webmanifest
Please commit your changes or stash them before you merge.
Aborting
⠴ Updating Open WebUI (Patience)
[ERROR] in line 32: exit code 0: while executing command output=$(git pull --no-rebase)
```

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 🔄 Steps to reproduce the issue.

Run the update command in either silent mode or verbose mode. I am currently on v0.5.19.

### ❌ Paste the full error output (if available).

⠼ Updating Open WebUI (Patience)error: Your local changes to the following files would be overwritten by merge:
        backend/open_webui/static/site.webmanifest
Please commit your changes or stash them before you merge.
Aborting
⠴ Updating Open WebUI (Patience)
[ERROR] in line 32: exit code 0: while executing command output=$(git pull --no-rebase)

### 🖼️ Additional context (optional).

_No response_

--- a/ct/openwebui.sh
+++ b/ct/openwebui.sh
@@ -44,7 +44,7 @@ function update_script() {
   $STD npm run build
   cd ./backend
   $STD pip install -r requirements.txt -U
-  cp -rf /opt/open-webui-backup/* /opt/open-webui/backend/data
+  cp -rf /opt/open-webui-backup/* /opt/open-webui/backend
   $STD git stash pop
   systemctl start open-webui.service
   msg_ok "Updated Successfully"

--- a/ct/openwebui.sh
+++ b/ct/openwebui.sh
@@ -29,6 +29,8 @@ function update_script() {
   fi
   msg_info "Updating ${APP} (Patience)"
   cd /opt/open-webui
+  mkdir /opt/open-webui/backup
+  cp -rf /opt/open-webui/backend/data /opt/open-webui/backup
   $STD git reset --hard
   output=$(git pull --no-rebase)
   if echo "$output" | grep -q "Already up to date."; then
@@ -41,6 +43,7 @@ function update_script() {
   $STD npm run build
   cd ./backend
   $STD pip install -r requirements.txt -U
+  cp -rf /opt/open-webui/backup/* /opt/open-webui/backend/data
   systemctl start open-webui.service
   msg_ok "Updated Successfully"
   exit

--- a/ct/openwebui.sh
+++ b/ct/openwebui.sh
@@ -29,6 +29,7 @@ function update_script() {
   fi
   msg_info "Updating ${APP} (Patience)"
   cd /opt/open-webui
+  $STD git reset --hard
   output=$(git pull --no-rebase)
   if echo "$output" | grep -q "Already up to date."; then
     msg_ok "$APP is already up to date."

--- a/ct/openwebui.sh
+++ b/ct/openwebui.sh
@@ -31,6 +31,7 @@ function update_script() {
   cd /opt/open-webui
   mkdir /opt/open-webui-backup
   cp -rf /opt/open-webui/backend/data /opt/open-webui-backup
+  git add -A
   $STD git stash
   $STD git reset --hard
   output=$(git pull --no-rebase)

--- a/ct/openwebui.sh
+++ b/ct/openwebui.sh
@@ -31,6 +31,7 @@ function update_script() {
   cd /opt/open-webui
   mkdir /opt/open-webui-backup
   cp -rf /opt/open-webui/backend/data /opt/open-webui-backup
+  $STD git stash
   $STD git reset --hard
   output=$(git pull --no-rebase)
   if echo "$output" | grep -q "Already up to date."; then
@@ -44,6 +45,7 @@ function update_script() {
   cd ./backend
   $STD pip install -r requirements.txt -U
   cp -rf /opt/open-webui-backup/* /opt/open-webui/backend/data
+  $STD git stash pop
   systemctl start open-webui.service
   msg_ok "Updated Successfully"
   exit

--- a/ct/openwebui.sh
+++ b/ct/openwebui.sh
@@ -29,8 +29,8 @@ function update_script() {
   fi
   msg_info "Updating ${APP} (Patience)"
   cd /opt/open-webui
-  mkdir /opt/open-webui/backup
-  cp -rf /opt/open-webui/backend/data /opt/open-webui/backup
+  mkdir /opt/open-webui-backup
+  cp -rf /opt/open-webui/backend/data /opt/open-webui-backup
   $STD git reset --hard
   output=$(git pull --no-rebase)
   if echo "$output" | grep -q "Already up to date."; then
@@ -43,7 +43,7 @@ function update_script() {
   $STD npm run build
   cd ./backend
   $STD pip install -r requirements.txt -U
-  cp -rf /opt/open-webui/backup/* /opt/open-webui/backend/data
+  cp -rf /opt/open-webui-backup/* /opt/open-webui/backend/data
   systemctl start open-webui.service
   msg_ok "Updated Successfully"
   exit

